src/index.ts

```diff
@@ -34,12 +36,31 @@ declare module 'express-session' {
     interface SessionData {
         siwe: SiweMessage;
         nonce: string;
-        ens: string;
+        // ens: string;
     }
 }
 
 const app = Express();
 
+// Setup ssx instance
+const ssx = new SSXServer({
+    signingKey: process.env.SSX_SIGNING_KEY,
+    providers: {
+        rpc: {
+            service: SSXRPCProviders.SSXInfuraProvider,
+            network: SSXInfuraProviderNetworks.MAINNET,
+            apiKey: process.env.INFURA_API_KEY ?? "",
+        },
+        sessionConfig: {
+            store: () => {
+                return new FileStoreStore({
+                    path: Path.resolve(__dirname, '../db/sessions'),
+                });
+            }
+        },
```

Comment:
sessionConfig takes the same input as Session and uses it under the hood to configure and establish a session store